src/include/lean/lean.h

```diff
@@ -451,6 +451,19 @@ static inline void lean_dec(lean_object * o) { if (!lean_is_scalar(o)) lean_dec_
 /* Just free memory */
 LEAN_EXPORT void lean_dealloc(lean_object * o);
 
+/*
+An object `o` satisfies `lean_is_ctor` if it is a member of an inductive type, barring the exceptions
+listed at https://lean-lang.org/lean4/doc/dev/ffi.html?highlight=ffi#translating-types-from-lean-to-c.
+A notable example of an exception are inductive types where all constructors have no parameters (enumeration types). 
+A notable example of types which *do* satisfy `lean_is_ctor` are structure types which are represented
+as inductive types with a single constructor (whose parameters are the structure's fields).
```

Comment:
This is phrased as if it's an exception, but it really isn't, given the way the elaborator treats structures. What about:
```suggestion
Behind the scenes, structure types are inductive types with a single constructor (whose parameters are the structure's fields), so values of structure types typically satisfy `lean_is_ctor`.
```

Reply:
I like the phrasing better, but now there's a weird transition from the previous sentence. So perhaps we can just remove this sentence entirely?